packager/live_packager_export.cc

```diff
@@ -49,6 +54,14 @@ void livepackager_free(LivePackager_t lp) {
   delete lp;
 }
 
+void live_packager_status_free(LivePackagerStatus_t self) {
```

Comment:
We don't need a function to free the status since we don't need to dynamically allocate it.